src/Listener/FailedStepListener.php

```diff
@@ -57,14 +52,12 @@ class FailedStepListener implements EventSubscriberInterface
      * @param Mink $mink
      * @param string $logDirectory
      * @param boolean $screenshot
```

Comment:
As I read horizontally, not vertically, IMHO it has no sense to format these params like a table :)